server/remote/common/status.go

```diff
@@ -2,31 +2,32 @@ package common
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/woodpecker-ci/woodpecker/server"
 	"github.com/woodpecker-ci/woodpecker/server/model"
 )
 
 func GetBuildStatusContext(repo *model.Repo, build *model.Build, proc *model.Proc) string {
-	name := server.Config.Server.StatusContext
-
+	ctx := server.Config.Server.StatusContextFormat
```

Comment:
What do you think about using GOs template function instead?

Reply:
https://pkg.go.dev/text/template

Reply:
Either text template or https://pkg.go.dev/os#Expand style imo

Reply:
@qwerty287 u may want to look into this ;) ?